### PR TITLE
Sarkars/fusedmatmul jimin

### PR DESCRIPTION
**Pull request by sayantan-nervana**
related to this: https://github.com/tensorflow/ngraph-bridge/pull/59

Added appropriate UTs and support for fused activation wrt PR 59

### CODE REVIEW

File: test/python/test_fusedMatMul.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class TestFusedMatMul(NgraphTest):
+    @pytest.mark.parametrize(
+        ("filename",),
+        (
+            ('fusedmatmul_0.pbtxt',),  # Relu
+            ('fusedmatmul_1.pbtxt',),  # Relu6
+            ('fusedmatmul_2.pbtxt',),  # No activation
+        ))
```
Comment (sayantan-nervana) [inline test/python/test_fusedMatMul.py]: gtest: `_FusedMatMul` is an internal op. Not sure if its Cpp exists.
Normal python test. writing a test with `tf.matmul` replaces it with `BatchMatMul` in cpp world, which does not fuse to `_FusedMatMul`.
Python test with pbtxt: Hence using pbtxts to read graphs with `_FusedMatMul` present in them

Comment (jiminha) [reply test/python/test_fusedMatMul.py]: I thought when it was 2x2 matrix, it doesn't go to BatchMatMul, but comes to FusedMatMul. Did you check?

Comment (sayantan-nervana) [reply test/python/test_fusedMatMul.py]: I checked, the logs show :
```
OP_placement:   Host    _SOURCE (NoOp)
OP_placement:   Host    _SINK (NoOp)
OP_placement:   Host    Placeholder_5 (Placeholder)
OP_placement:   Host    Placeholder_4 (Placeholder)
OP_placement:   Host    Placeholder_3 (Placeholder)
OP_placement:   Host    BiasAdd_1 (IdentityN)
OP_placement:   nGraph[0]       MatMul_1 (BatchMatMul)
OP_placement:   nGraph[0]       BiasAdd_1_ngraph/_0 (BiasAdd)
```

even for `2x2` it seems to generate `BatchMatMul`. Hence removed the python based graph construction and inserted a `pbtxt` based test

File: src/ngraph_builder.cc [hunk 2]
```diff
@@ -1916,6 +1916,97 @@ static Status TranslateFusedBatchNormGradOp(
...
+    ng_dot = ConstructNgNode<ngraph::op::Dot>(op->name(), ng_lhs, ng_rhs);
+
+    return Status::OK();
+  };
+
+  // TODO: Need to add BiasAdd + Activation fusedMatMul op
+  /*if (!VecStrCmp(fused_ops, {"BiasAdd"})) {
+    return errors::Unimplemented("Unsupported _FusedMatMul " +
+                                 str_util::Join(fused_ops, ","));
+  }*/
```
Comment (jiminha) [inline src/ngraph_builder.cc]: Remove this since you now have Relu support?

Comment (sayantan-nervana) [reply src/ngraph_builder.cc]: Done 👍

**Pull request merged**